transformation/transformationPlugins/jsonMessageTransformers/jsonTypeMappingsSanitizationTransformerProvider/src/test/java/org/opensearch/migrations/replay/TypeMappingsSanitizationProviderTest.java

```diff
@@ -87,39 +91,43 @@ public void testSimpleTransform() throws JsonProcessingException {
     }
 
     @Test
+    @Disabled("I'm not sure what Greg was intending to test with this")
```

Comment:
The "Not" in the name is confusing - this is to test that w/out types and with the latest source, the PUT index request should be returned untouched.